src/main/resources/db/migration/changes/DiningCommons.json

```diff
@@ -0,0 +1,86 @@
+{ "databaseChangeLog": [
```

Comment:
Why is this here? What end user functionality does it support?

There is no issue among the five approved issues that describes creating an entity for a DiningCommons (unless it is an oversight, in which case please point it out.)

This should be removed.